compiler/resolution/wrappers.cpp

```diff
@@ -477,17 +477,19 @@ buildDefaultWrapper(FnSymbol* fn,
 
 FnSymbol*
 defaultWrap(FnSymbol* fn,
-            Vec<ArgSymbol*>* actualFormals,
+            std::map<int, ArgSymbol*>* actualFormals,
```

Comment:
There is a popular convention to define functions such that inputs come first and then outputs
come last.  If there are in/outs and out then I think it is common to place the in/outs before the
outs.  I don't think it is necessary to "fix" this function for this PR but something to keep in mind?